tests/integration/targets/azure_rm_containerinstance/tasks/main.yml

```diff
@@ -143,6 +143,95 @@
       - output.containerinstances[1]['ports'] != None
       - output.containerinstances[1]['containers'] != None
 
+- name: Create sample container instance with volume
+  azure_rm_containerinstance:
+    resource_group: "{{ resource_group }}"
+    name: myContainerInstanceGroup
+    os_type: linux
+    ip_address: public
+    location: eastus
+    ports:
+      - 80
+    containers:
+      - name: mycontainer1
+        image: httpd
+        memory: 1.5
+        volume_mounts:
+          - name: filesharevolume
```

Comment:
How did this account name ? It can't access?

Reply:
Hi @Fred-sun Apologies, I didn't get your question. Could you please elaborate your question so that I can understand it better and take the action? Thank you!
FYI - I am referring to this document for Azure file volumes - https://docs.microsoft.com/en-us/azure/container-instances/container-instances-volume-azure-files

On line no. 165 volumes specification is initialized and there we have specified three parameters (_storage_account_name_, _share_name_ and _storage_account_key_) which are needed in order to mount azure file volume on container instance

Reply:
Hi @Fred-sun Could you please guide me here to understand what changes are required for line no. 160? Thanks for your help!